jwst/ami/instrument_data.py

```diff
@@ -331,11 +332,19 @@ def reset_nwav(self, nwav):
 
     def mast2sky(self):
         """
-        Rotate hole center coordinates:
-            Clockwise by the V3 position angle - V3I_YANG from north in degrees if VPARITY = -1
-            Counterclockwise by the V3 position angle - V3I_YANG from north in degrees if VPARITY = 1
+        Rotate hole center coordinates.
+
+        Rotation of coordinates is:
+            Clockwise by the ROLL_REF + V3I_YANG from north in degrees if VPARITY = -1
+            Counterclockwise by the ROLL_REF + V3I_YANG from north in degrees if VPARITY = 1
         Hole center coords are in the V2, V3 plane in meters.
 
+        Notes
+        -----
+            Nov. 2024 email discussion with Tony Sohn, Paul Goudfrooij confirmed V2/V3 coordinate
+            rotation back to "North up" equatorial orientation should use ROLL_REF + V3I_YANG
+            (= PA_APER).
```

Comment:
Super tiny, but this section still needs to be un-indented, to match the "Notes" indent level.